benchmark/parse_output.py

```diff
@@ -117,6 +117,32 @@ def parse_darkhorse(input_f, low_lpi=0.0, high_lpi=0.6, return_genomes=False):
     return None
 
 
+def parse_hgtector(input_f):
+    """ Parse output of HGTector version 0.2.1.
+
+    Parameters
+    ----------
+    input_f: string
+        HGTector working directory path
+
+    Returns
+    -------
+    output: list
+        list of putative HGTs
+    """
+    reading = 0
+    output = []
+    for line in input_f:
+        line = line.rstrip('\r\n')
+        if not line:
+            continue
+        elif line.startswith('Putatively HGT-derived genes:'):
+            reading = 1
+        elif reading:
+            output.append(line.split('\t')[0])
```

Comment:
Just Gene IDs. Previously I had the code return all fields. In order to simplify the unit test I trimmed that down to gene IDs only (so that I didn't have to append a long output_exp value in the code).